routes/admin.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+
+const adminController = require('../controllers/admin')
+
+const router = express.Router();
+
+// /admin/add-product => GET
+router.get('/add-product', adminController.getAddProduct);
+
+router.get('/products', adminController.getProducts);
+
+// /admin/add-product => POST
+router.post('/add-product', adminController.postAddProduct);
+
+router.get('/edit-product/:productId', adminController.getEditProduct);
+
+router.post('/edit-product', adminController.postEditProduct);
+
+router.post('/delete-product', adminController.postDeleteProduct);
```

Comment:
no need to add words like `delete` or `add` or `edit` in the route. as you are using proper HTTP methods like `post` or `get`. then it indicates what is happening.